deployment-platforms/render/src/index.js

```diff
@@ -0,0 +1,149 @@
+import express from 'express'
+import bodyParser from 'body-parser'
+import { PrismaClient } from '@prisma/client'
+
+const prisma = new PrismaClient()
+const app = express()
+
+app.use(bodyParser.json())
+app.use(express.static('public'))
+
+app.get(`/api`, async (req, res) => {
+  res.json({ up: true })
+})
+
+app.get(`/api/seed`, async (req, res) => {
```

Comment:
interesting! We also have seed functionality built into Prisma ORM: https://www.prisma.io/docs/orm/prisma-migrate/workflows/seeding

Did that not work in this case?

Reply:
Thanks, I'll update this.

I didn't know about the seed feature. (I actually copied the existing Heroku example to keep these two examples easier to compare.)

Reply:
Ah, that makes sense. I think the heroku example is a bit dated. Understand why you went this path though 🙏

Reply:
Just updated the code with a seed script. Also added a "Seed the database" section to the docs side. :)